app/services/analytics_events.rb

```diff
@@ -6304,6 +6304,20 @@ def oidc_logout_visited(
     )
   end
 
+  # Tracks when user lands on page notifying them multiple profiles contain same information
+  def one_account_multiple_accounts_detected
```

Comment:
Similar to above, `one_account_duplicate_profiles_detected`. Similarly below: `one_account_all_duplicate_profiles_confirmed` and ``one_account_unknown_duplicate_profiles_confirmed`?